### ISSUE TITLE
Keyboard configuration on install page GUI

### ISSUE THREAD

**Issue by MarcoEnxuto** [microsoft/CBL-Mariner#1977]
Please provide on the graphic installer an option to select the keyboard setting.
I had to configure the keyboard to the Portuguese layout by downloading the following packages.
kbd and nano to open the sysconfig/console file.

For those who are interested:
tdnf install kbd
Localectl set-keymap **pt-latin1** (this sets the keyboard layout instantly)

To persist on the system, do the following
tdnf install nano (you can use vi, but i prefer nano)
nano\vi /etc/sysconfig/console
uncoment keymap and set is value to **pt-latin1**

**Comment by ohault** [comment]
+1

**Comment by MarcoEnxuto** [comment]
Hi, is this being considered? 😊

**Comment by Malateshk007** [comment]
@MarcoEnxuto , I hope this message finds you well. 
Thank you for reaching out to us two years ago regarding your request. We appreciate your patience.

Given the extended period since your initial inquiry, we kindly ask if you still require the support or if your circumstances have changed. If you no longer need assistance, please feel free to let us know, and we will close this request accordingly.

Your feedback is valuable to us, and we want to ensure that we provide the best service possible. If there are any other queries or concerns, please don’t hesitate to reach out.

Thank you for your understanding, and we look forward to hearing from you.

**Comment by MarcoEnxuto** [comment]
Yes, I think it's still a bad experience not having the choice on the setup. Thanks for replying.

**Comment by jperrin** [comment]
administratively closing
